tests/org.eclipse.ui.tests/Eclipse UI Tests/org/eclipse/ui/tests/api/IWorkbenchPageTest.java

```diff
@@ -499,10 +499,10 @@ public void testOpenEditor4() throws Throwable {
 	@Test
 	public void testGet_SetEditorAreaVisible() throws Throwable {
 		fActivePage.setEditorAreaVisible(true);
-		assertTrue(fActivePage.isEditorAreaVisible() == true);
+		assertTrue(fActivePage.isEditorAreaVisible());
 
 		fActivePage.setEditorAreaVisible(false);
-		assertTrue(fActivePage.isEditorAreaVisible() == false);
+		assertFalse(fActivePage.isEditorAreaVisible());
```

Comment:
Dito: automatic clean-up? (it's OK though)

Reply:
Manual cleanup to reduce accidental complexity of `assertTrue(!condition)`.